### PR TITLE
Replace Token UI with Inline Card

### PR DESCRIPTION
**Pull request by jubalm**
- [x] implement warning icon in inline card component
- [x] copy status as tooltip
- [x] make copy optional

| Screenshot 2024-10-22 at 3 44 56 PM |
|:--:|
| * token amount * |

| Screenshot 2024-10-31 at 9 42 57 AM |
|:--:|
| *token with warning* |

| Screenshot 2024-10-31 at 9 46 58 AM |
|:--:|
| *copy confirmation* |

Update:
- [x] fix hover background not covering the entire width
- [x] fix token symbol and value amount are far apart on swap ui

| image | Screenshot 2024-11-05 at 1 40 04 PM |
|--|--|
| *from discord report* | *changed to center aligned* |

### CODE REVIEW

**Comment by KillariDev** [comment]
image
I noticed when currencies are overed, some extra background appears

**Review by KillariDev — APPROVED** [review]

File: app/css/interceptor.css [hunk 1]
```diff
@@ -1584,35 +1584,36 @@ header:has(form[role='search']) h1 {
...
+	--bg-color: var(--card-bg-color);
+	--button-color: #77738ccc;
 	--button-color-hilite: white;
-	--status-backdrop-color: #303030ed; /* opaque black */
-	--status-success-color: #4fb64f;	/* vivid green */
 	--bg-inline-padding: 4px;
 	--bg-block-padding: 1px;
 	--image-size: 1em;
 	--min-text-width: 5.5ch;
+	--status-backdrop-color: #303030ed;
```
[the remark this answers is not in the document]
Comment (jubalm) [reply app/css/interceptor.css:1596]: same tint but have some transparency

File: app/ts/components/subcomponents/Tooltip.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+			popoverRef.current?.hidePopover()
+			return
+		}
+
+		popoverRef.current?.showPopover()
+		setTimeout(() => config.value = undefined, config.value.duration || 1500)
+	})
+
+	return (
+		<div ref={ popoverRef } class='tooltip' popover style={ { left: config.value?.x || 0, top: config.value?.y || 0 } }>
```
Comment (KillariDev) [inline app/ts/components/subcomponents/Tooltip.tsx:25]:
```suggestion
		<div ref = { popoverRef } class='tooltip' popover style = { { left: config.value?.x || 0, top: config.value?.y || 0 } }>
```

File: app/ts/components/subcomponents/coins.tsx [hunk 3]
```diff
@@ -128,44 +128,21 @@ export function TokenSymbol(param: TokenSymbolParams) {
...
-							<img src = '../img/rename.svg'/>
-						</span>
-					</button>
-				</> }
-			</span>
-		</span>
-	</span>
+	return <>
+		<TokenIdOrNameOrNothing { ...param } />
+		<InlineCard icon = { generateIcon } copyValue = { tokenAddressString } label = { tokenName } onEditClicked = { () => param.renameAddressCallBack(param.tokenEntry) } warningMessage = { warningMessage } style = {{ ...defaultCardStyles, ...param.style }} />
```
Comment (KillariDev) [inline app/ts/components/subcomponents/coins.tsx:144]:
```suggestion
		<InlineCard icon = { generateIcon } copyValue = { tokenAddressString } label = { tokenName } onEditClicked = { () => param.renameAddressCallBack(param.tokenEntry) } warningMessage = { warningMessage } style = { { ...defaultCardStyles, ...param.style } } />
```

**Pull request merged**